Add governance hooks for identity assertion (#267)
creator-assertions/identity-assertion: docs/modules/ROOT/partials/introduction/scope.adoc

This specification draws upon and extends existing work from the https://c2pa.org[Coalition for Content Provenance and Authenticity (C2PA)], which has developed a link:https://spec.c2pa.org/specifications/specifications/2.4/specs/C2PA_Specification.html[technical specification] for providing content provenance and authenticity.

It is designed to enable global, opt-in, adoption of digital provenance techniques through the creation of a rich ecosystem of digital provenance enabled applications for a wide range of individuals and organizations while meeting appropriate security requirements.

The C2PA technical specification allows other standards bodies to define additional metadata that can be incorporated into a _<<C2PA Manifest>>._
These statements come in the form of a _<<_c2pa_assertion,C2PA assertion>>._

> This specification describes a specific _<<C2PA assertion>>_ that can be added to a _<<C2PA Manifest>>_ to allow an _<<_actor,actor>>_ backed by a governing authority that manages its own trust registry to prove control over a digital identity and to bind that identity to a _<<_c2pa_asset,C2PA asset>>_ (and, at their option, one or more _<<_c2pa_assertion,C2PA assertions>>_) produced by them or on their behalf.

Reviewer: ```suggestion
This specification describes a specific _<<C2PA assertion>>_ that can be added to a _<<C2PA Manifest>>_ to allow an _<<_actor,actor>>_ optionally backed by a governing authority that manages its own trust registry to prove control over a digital identity and to bind that identity to a _<<_c2pa_asset,C2PA asset>>_ (and, at their option, one or more _<<_c2pa_assertion,C2PA assertions>>_) produced by them or on their behalf.
```

`trust_registry` is optional but this scopes the spec to an actor backed by a governing authority that manages its own trust registry.  That reads as presuming governance for every actor and excluding the ungoverned base (1.2) case.  Suggest framing governance as an optional layer.

> This specification describes a specific _<<C2PA assertion>>_ that can be added to a _<<C2PA Manifest>>_ to allow an _<<_actor,actor>>_ backed by a governing authority that manages its own trust registry to prove control over a digital identity and to bind that identity to a _<<_c2pa_asset,C2PA asset>>_ (and, at their option, one or more _<<_c2pa_assertion,C2PA assertions>>_) produced by them or on their behalf.

Reviewer 2: ```suggestion
This specification describes a specific _<<C2PA assertion>>_ that can be added to a _<<C2PA Manifest>>_ to allow an _<<_actor,actor>>_ to prove control over a digital identity and to bind that identity to a _<<_c2pa_asset,C2PA asset>>_ (and, at their option, one or more _<<_c2pa_assertion,C2PA assertions>>_) produced by them or on their behalf. These actors can be backed by an authority governs the actor or how the actor's credential is issued.
```

Reviewer 3: One actor may be be backed by multiple governing authorities, and may have multiple digital identities they could use (each one backed by a different governing authority); perhaps we should be add some language on how the digital identity itself is recognized by a the governing authority.

> It further allows a _<<_named_actor,named actor>>_ to demonstrate membership in or affiliation with an ecosystem which creates and manages its own _<<_governance_framework,governance framework>>._

Reviewer 2: An actors use of a credential issued by an issuer on a trust registry might demonstrate affiliation with the governance framework of those issuers, or it might not. If we assume something like the Mozilla Root Program will be allowed in this framework, there's a much weaker link between the actor and the Root Program governance.

> The *<<_identity_assertion,identity assertion>>* can be complemented through the use of a _<<_referenced_registry,referenced registry>>_ which allows for a link to additional assertions can be made by the content creator outside of the _<<_c2pa_asset,C2PA asset>>_ or a _<<_referenced_registry,referenced registry>>_ which contains metadata elements to be verified from the metadata claims made on the _<<_c2pa_asset,C2PA asset>>_.

Reviewer: This sentence is hard to parse and conflates two functions. 'allows for a link to additional assertions can be made' is grammatically broken, and the two distinct uses, linking external assertions versus verifying metadata, are run together. Suggest splitting into two sentences, one per function.
